hack/generate/changelog/util/fragment.go

```diff
@@ -28,13 +28,13 @@ func (f *Fragment) Validate() error {
 }
 
 type FragmentEntry struct {
-	Description string          `yaml:"description"`
-	Kind        EntryKind       `yaml:"kind"`
-	Breaking    bool            `yaml:"breaking"`
-	Migration   *EntryMigration `yaml:"migration,omitempty"`
-	PullRequest *uint           `yaml:"pull_request_override,omitempty"`
+	Description string          `json:"description"`
```

Comment:
the 'yaml' to 'json' change in the annotations, is that expected?

Reply:
Yep! The `sigs.k8s.io/yaml` library [actually converts the YAML to JSON and then unmarshals as JSON](https://github.com/kubernetes-sigs/yaml/blob/9fc95527decd95bb9d28cc2eab08179b2d0f6971/yaml.go#L47-L64).